clang/lib/Interpreter/RemoteJITUtils.cpp

```diff
@@ -91,7 +95,8 @@ createSharedMemoryManager(SimpleRemoteEPC &SREPC,
 
 Expected<std::unique_ptr<SimpleRemoteEPC>>
 launchExecutor(StringRef ExecutablePath, bool UseSharedMemory,
-               llvm::StringRef SlabAllocateSizeString) {
+               llvm::StringRef SlabAllocateSizeString,
+               std::function<void()> CustomizeFork) {
```

Comment:
Why not doing something like:
```suggestion
               std::function<void()> InitChild = [](){/*move the code from `if (Child)` into a default lambda here...*/}) {
```

Reply:
I would say the current design is better. Because the lambda actually does not need to do anything by default.
We only need redirections for xeus-cpp, and xeus-cpp can set the lambda to do that.